spec/views/users/show.html.erb_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'users/show.html.erb', type: :view do
+  let(:user) { FactoryBot.create :user }
+
+  before do
+    assign(:user, user)
```

Comment:
does this inject our local :user into the database for the test or why do we need this?

Reply:
You are right, should actually be enough to just build the user! Good catch

Reply:
Actually we still need create, because we access the users_path which is filled with data from the db. I can explain it to you in person later on.